### PR TITLE
Applets: Add a network graph applet

### PR DESCRIPTION
**Pull request by kleinesfilmroellchen**
This resource graph applet shows incoming network traffic graphically, and the tooltip shows both TX and RX. The data of all adapters is combined. There's even automatic scaling!

image

### CODE REVIEW

**Comment by linusg** [comment]
Mind adding a screenshot? :)

File: Userland/Applets/ResourceGraph/main.cpp [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 #include <AK/CircularQueue.h>
 #include <AK/JsonObject.h>
+#include <AK/Random.h>
```
Comment (linusg) [inline Userland/Applets/ResourceGraph/main.cpp:10]: What's this for?

Comment (kleinesfilmroellchen) [reply Userland/Applets/ResourceGraph/main.cpp:10]: Ask clang-tidy :^)

**Comment by bgianfo** [comment]
I like it, Picasso!

**Pull request merged**